swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -63,14 +52,13 @@ Follow the instructions given in this section to develop the service.
         "fee": 7000
     }
     ```
+   
+    ![Define a record](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_record.gif)
 
-    1. Copy the sample JSON payload.
-    2. Open the VS Code [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette).
-    3. Search and select the `Ballerina: Paste JSON as record` command.
-
-    ![Paste JSON as record](/learn/images/integration-tutorials/sending-a-message-to-a-service/paste_json_as_record.gif)
+    > **Note:**
+    > While it's possible to work with JSON payloads directly, utilizing record types in your code offers several advantages. Records provide enhanced type safety, validation capabilities, and automated code generation, and are maintainable when handling structured data.
 
-5. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, retrieves relevant details from the backend service, and responds to the request.
+4. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, retrieves relevant details from the backend service, and responds to the request.
 
     - Open [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code
    
```

Comment:
"Define an HTTP resource that allow ..."

We are still not mentioning the return types?